### PR TITLE
Localize fixed strings and remove not used strings

### PR DESCRIPTION
**Pull request by magol**
New strings:
```JSON
"country-panel": {
    "lowcarbon": "Low-carbon",
    "renewable": "Renewable"
},
"country-history": {
    "electricityorgin24h": "Origin of electricity in the last 24 hours",
    "electricityprices24h": "Electricity prices in the last 24 hours",
    "Getdata": "Get historical data and forecast API"
},
"tooltips": {
    "lowcarbon": "Low-carbon",
    "renewable": "Renewable"
},
"misc": {
    "pro": "pro"
}
```

### CODE REVIEW

File: web/locales/sv.json [hunk 1]
```diff
@@ -48,8 +50,8 @@
         "carbonintensity": "Utsläpp av koldioxid"
     },
     "tooltips": {
-        "electricityprice": "Elpris (nästa dag)",
-        "fossilfuels": "fossila bränslen",
+        "lowcarbon": "CO2-neutralt",
```
Comment (corradio) [inline web/locales/sv.json]: The translation shouldn't be "co2 neutral", it should be "low carbon" (which is not the same)

Comment (magol) [reply web/locales/sv.json]: Yes, i know. But i could not figurer out a correct translation that works in Swedish.

What is the definition of "low carbon"? How do you calculate it?
Can i use "fossil-free" instead?

Comment (corradio) [reply web/locales/sv.json]: Low carbon is indeed the same as fossil-free. In Danish it's hard to find an equivalent of "low-carbon", but "fossilfri" works fine, so I like your suggestion.

Comment (magol) [reply web/locales/sv.json]: Great  :-)
I have fix that now

**Pull request merged**